routing/router.go

```diff
@@ -2015,10 +2015,7 @@ func (r *ChannelRouter) sendPayment(
 
 	// We'll also fetch the current block height so we can properly
 	// calculate the required HTLC time locks within the route.
-	_, currentHeight, err := r.cfg.Chain.GetBestBlock()
-	if err != nil {
-		return [32]byte{}, nil, err
-	}
+	currentHeight := int32(atomic.LoadUint32(&r.bestHeight))
```

Comment:
seems like we cannot guarantee the `bestHeight` is up to date when this method is called.

Reply:
Is this the same comment as what @guggero mentions above or different?